nbbuild/antsrc/org/netbeans/nbbuild/RatReportTask.java

```diff
@@ -132,9 +132,9 @@ public void execute() throws BuildException {
         Set<String> clusterList = new TreeSet<>();
         for (String key : getProject().getProperties().keySet()) {
             if (key.startsWith("nb.cluster.")) {
-                String simplfiedKey = key.replaceAll("nb.cluster.", "");
-                simplfiedKey = simplfiedKey.replaceAll(".dir", "");
-                simplfiedKey = simplfiedKey.replaceAll(".depends", "");
+                String simplfiedKey = key.replace("nb.cluster.", "");
```

Comment:
Not completely equivalent, but *probably* better in the new code again.

Reply:
Now, the string "nb.cluster" is exactly matched instead of "nb<any_character_here>cluster<any_character_here>"